src/modules/airdrop/module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from "@nestjs/common";
+import { TypeOrmModule } from "@nestjs/typeorm";
+
+import { UserModule } from "../user/module";
+
+import { WalletRewardsFixture } from "./adapter/db/wallet-rewards-fixture";
+import { CommunityRewardsFixture } from "./adapter/db/community-rewards-fixture";
+import { AirdropService } from "./services/airdrop-service";
+import { AirdropController } from "./entry-points/http/controller";
+
+import { CommunityRewards } from "./model/community-rewards.entity";
+import { WalletRewards } from "./model/wallet-rewards.entity";
+import { Deposit } from "../scraper/model/deposit.entity";
+
+@Module({
+  providers: [AirdropService, WalletRewardsFixture, CommunityRewardsFixture],
```

Comment:
The fixtures `WalletRewardsFixture` and `CommunityRewardsFixture` are only used for the e2e tests, right? I think it would make sense to only load them when running the tests in the future. Wdyt?

Reply:
Yea exactly. I guess that overlaps with the mode splitting work I am currently looking into.

Reply:
Yes. It is a good idea 👍

Reply:
we can have `public-api`, `scraper` and `test` as running modes, execution modes or whatever name it'll have 😄